app/models/missing_link_finder.rb

```diff
@@ -1,3 +1,11 @@
+require "capybara/mechanize"
```

Comment:
I recommend viewing this file in side-by-side diff mode, since it changed significantly. It used to use simple `Net::HTTP` requests but now spins up a (non-Javascript) Capybara session, which lets us easily find elements like the iframes and frames (and next/prev links), and adds a useful concept of "the current page".

Reply:
Hah, I was having trouble reading it, then I did this very thing. :high-five: